data/Language/TinyBasic.yaml

```diff
@@ -0,0 +1,28 @@
+identifier: TinyBasic
+full_name: TinyBasic
+versions: 1.0, 2.0
```

Comment:
discard this.  We dont have versions in the schema yet.  There is a separate task for creating properties ;-)

Reply:
Gotcha, fixed!